fondat/aws/bedrock/resources/agent.py

```diff
@@ -0,0 +1,262 @@
+"""Resource for managing a specific Bedrock agent."""
+
+from collections.abc import Iterable
+from typing import Any, Mapping
+
+from fondat.aws.client import Config, wrap_client_error
+from fondat.resource import resource
+from fondat.security import Policy
+
+from ..clients import agent_client, runtime_client
+from ..decorators import operation
+
+from .versions import VersionsResource
+from .aliases import AliasesResource
+from .action_groups import ActionGroupsResource
+from .collaborators import CollaboratorsResource
+from .flows import FlowsResource
+from .sessions import SessionsResource
+from .memory import MemoryResource
+
+
+@resource
+class AgentResource:
+    """
+    Resource for managing a specific Bedrock agent.
+    Provides access to agent metadata, versions, aliases, action groups, and runtime operations.
+    """
+
+    __slots__ = ("_id", "config_agent", "config_runtime", "policies")
+
+    def __init__(
+        self,
+        agent_id: str,
+        *,
+        config_agent: Config | None = None,
+        config_runtime: Config | None = None,
+        policies: Iterable[Policy] | None = None,
+    ):
+        self._id = agent_id
+        self.config_agent = config_agent
+        self.config_runtime = config_runtime
+        self.policies = policies
+
+    @operation(method="get", policies=lambda self: self.policies)
+    async def get(self) -> Mapping[str, Any]:
```

Comment:
We should probably create a new class : `Agent` and return that from the resource.